### DISCUSSION TITLE
A mix of questions and ideas/suggestions

### DISCUSSION THREAD

**Discussion by gewhere** [tidalcycles/strudel#545] [Q&A] [answered]
1. Is there multi-channel expansion on Strudel?
2. Is Tidal's function `once` available on Strudel?
3. Is this `|` the logical OR?
4. Implement a record button, or a function (for audio recordings)?
5. Implement a keyboard shortcut to get redirected to the help lemma on the bottom page?  (Similar to Cmd+D on SC3)

**Comment by yaxu** [comment, marked as answer]
* Is there multi-channel expansion on Strudel?

As far as I understand it, yes this is fundamental to how tidal and strudel works

* Is Tidal's function once available on Strudel?

No

* Is this | the logical OR?

In mininotation, no, it denotes random selection

**Comment by felixroos** [comment]
> Implement a record button, or a function (for audio recordings)?

it could take a while for that feature to land.. nowadays it's pretty easy to record your desktop with external programs, e.g. audacity, see https://support.audacityteam.org/basics/recording-desktop-audio

> Implement a keyboard shortcut to get redirected to the help lemma on the bottom page? (Similar to Cmd+D on SC3)

yep something like that will happen. autocomplete with documentation is also almost there
